lib/blueprinter/extractors/auto_extractor.rb

```diff
@@ -2,7 +2,16 @@ module Blueprinter
   class AutoExtractor < Extractor
     def extract(field_name, object, local_options, options = {})
       extractor = object.is_a?(Hash) ? HashExtractor : PublicSendExtractor
-      extractor.extract(field_name, object, local_options, options)
+      extraction = extractor.extract(field_name, object, local_options, options)
+      options.key?(:datetime_format) ? format_datetime(extraction, options[:datetime_format]) : extraction
```

Comment:
I'm just wondering if this formatting should be here or it should be in it's own extractor or maybe an entirely new class??

Reply:
Originally I had it as its own extractor, but it has to work for hashes as well and I didn't want to avoid the existing `HashExtractor` logic.

Reply:
I'm just thinking where best to put this. Now that `AutoExtractor` is both extracting and formatting.

Maybe formatting dates isnt an `Extractor` because it's not extracting. Maybe it could be a kind of helper module called `FormatHelpers` or a class called `Formatter` where you could do something like `Formatter.new(extraction, option).format`

Reply:
I can really get behind a `Formatter` class with a constructor API of something like `Formatter.format(extraction, options)`. Let it do it's own initialization if necessary 👍